pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -2218,7 +2247,7 @@ func (r *ClickHouseReader) SetTTL(ctx context.Context,
 				}
 				return
 			}
-			req += " SETTINGS distributed_ddl_task_timeout = -1"
```

Comment:
@srikanthccv Why are we removing this setting?

Reply:
Why did we add it in the first place?

Reply:
It doesn't take any time (because it would just be a metadata update) if we do not materialize(the change this PR proposes).

Reply:
Great then it's not required, I hope you have tested these changes on a large DB.